modules/lang-painless/spi/src/main/java/org/elasticsearch/painless/spi/WhitelistClass.java

```diff
@@ -49,7 +49,7 @@
     /**
      * Allow the Java class name to only be specified as the fully-qualified name.
      */
-    public final boolean onlyFQNJavaClassName;
+    public final boolean noImport;
```

Comment:
I know we've discussed this rename, but I would keep this as is until that change is ready, to keep consistency.